### DISCUSSION TITLE
data not getting mapped for xlsx file to pojo object

### DISCUSSION THREAD

**Discussion by Mayankt-eGov** [ozlerhakan/poiji#286] [Q&A]
i m using poiji for mapping data from xlsx file to my pojo . it seems to work in local but in my dev server it neither throw any exception nor read the data . Can you please help me out on this

**Comment by ozlerhakan** [comment]
Hi @Mayankt-eGov ,

What are the errors you're getting when running the app in the server? You might need to add more logging to the workflow.

**Reply by Mayankt-eGov** [reply]
Thats the issue , i have kept the code in try-catch block also but no error gets thrown , code just dont print any log from that point .
